K9 crash try attaching files

### Expected behaviour
Adding attachment and browising files whit Amaze or other files manager I would like to attach a file

### Actual behaviour
K9 crash when I select the files whit Amaze or other files manager (process "com.fsck.k9" stop); if I select files from stock image gallery app or stock audio library app it works fine.
Downgrading K9 to v5.008 everything works fine

### Steps to reproduce
1.Create new email
2.Add attachment
3.Select a file whit Amaze or any other files manager

### Environment
K-9 Mail version:
5.010
Android version:
4.0.4
Account type (IMAP, POP3, WebDAV/Exchange):
IMAP

```
05-29 09:47:58.000 E/AndroidRuntime( 9042): Caused by: java.lang.UnsatisfiedLinkError: Couldn't load os-compat: findLibrary returned null
05-29 09:47:58.000 E/AndroidRuntime( 9042): 	at java.lang.Runtime.loadLibrary(Runtime.java:365)
05-29 09:47:58.000 E/AndroidRuntime( 9042): 	at java.lang.System.loadLibrary(System.java:535)
05-29 09:47:58.000 E/AndroidRuntime( 9042): 	at de.cketti.safecontentresolver.Os.<clinit>(Os.java:21)
```

Looks like it's failing to find the native library that safe content resolver uses. 

What device are you using - maybe it's not running one the supported ABIs??

https://github.com/cketti/SafeContentResolver#native-code

--- a/k9mail/build.gradle
+++ b/k9mail/build.gradle
@@ -29,7 +29,7 @@ dependencies {
     compile 'de.cketti.library.changelog:ckchangelog:1.2.1'
     compile 'com.github.bumptech.glide:glide:3.6.1'
     compile 'com.splitwise:tokenautocomplete:2.0.7'
-    compile 'de.cketti.safecontentresolver:safe-content-resolver-v14:0.0.1'
+    compile 'de.cketti.safecontentresolver:safe-content-resolver-v14:0.9.0'
     compile 'com.github.amlcurran.showcaseview:library:5.4.1'
 
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.1'

--- a/k9mail/src/main/AndroidManifest.xml
+++ b/k9mail/src/main/AndroidManifest.xml
@@ -391,7 +391,13 @@
             android:exported="true"
             android:grantUriPermissions="true"
             android:multiprocess="true"
-            android:readPermission="${applicationId}.permission.READ_ATTACHMENT"/>
+            android:readPermission="${applicationId}.permission.READ_ATTACHMENT">
+            
+            <meta-data
+                android:name="de.cketti.safecontentresolver.ALLOW_INTERNAL_ACCESS"
+                android:value="true" />
+        
+        </provider>
 
         <provider
             android:name=".provider.MessageProvider"

